src/plugins/telemetry/public/components/opt_in_banner.test.tsx

```diff
@@ -10,15 +10,23 @@ import React from 'react';
 import { EuiButton } from '@elastic/eui';
 import { shallowWithIntl } from '@kbn/test-jest-helpers';
 import { OptInBanner } from './opt_in_banner';
+// eslint-disable-next-line @kbn/eslint/no-restricted-paths
+import { docLinksServiceMock } from '../../../../core/public/doc_links/doc_links_service.mock';
```

Comment:
You can use the top level `mocks` exported so you don't need the eslint disable comment:
```suggestion
import { docLinksServiceMock } from '../../../../core/public/mocks';
```